tarek.js

```diff
@@ -1 +1,2 @@
-var newFile= 1;
\ No newline at end of file
+var newFile= 1;
+var no=11;
```

Comment:
great code! naming is a bit confusing though

Reply:
righht!!!!